tinyms/serving/client/client.py

```diff
@@ -93,7 +93,7 @@ def list_servables():
         return 'Server not started'
 
 
-def predict(img_path, servable_name, dataset_name="mnist", strategy="TOP1_CLASS"):
+def predict(img_path, servable_name, dataset_name="mnist", strategy="TOP1_CLASS", servable_path=None, ckpt_path=None):
```

Comment:
Same comment with `predict` function, if users have specified the `servable_name`, then they don't need to specify the `servable_path` and `ckpt_path` at all. Or else users could get confused.

Reply:
Do you mean, remove the comments for these parameters?

Reply:
Yes, I don't think users need to specify the `servable_path` and `ckpt_path` manually. And in the later version, we are going to support `MINDIR` format.